### PR TITLE
[RED-000] update training configs

### PR DESCRIPTION
**Pull request by jonathonsather-bc**
* maintain image aspect ratio
* 6d rotation representation
* train for more epochs
* "pick sugar" task without rotations for experiment
* update tests

### CODE REVIEW

File: diffusion_policy/config/task/pick_sugar_image.yaml [hunk 1]
```diff
@@ -25,5 +25,5 @@ dataset:
   pad_before: 0
   pad_after: 0
   seed: 42
-  validation_ratio: 0.00
+  validation_ratio: 0.05
```
Comment (jonathonsather-bc) [inline diffusion_policy/config/task/pick_sugar_image.yaml:28]: can probably make 0 again, since validation loss quickly starts rising, even though overfitting does not seem to be occurring. Seems more a symptom of taking more confident actions that are reasonable, but different from the validation set
